app/models/loader/handlers/public_key.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Loader
+  module Handlers
+    # extends the policy loader to store public keys for users as variables. This happens
+    # after the initial policy load, since variable values are not part of the policy load
+    module PublicKey
+  
+      # When a public key is encountered in a policy, it is saved here. It can't be written directly into
```

Comment:
This module is for loading a pubic key, but it mentions the `credentials` table, which AFAIK, doesn't have anything to do with public keys.

Reply:
Sure, thanks for catching that. I will clean up the comments.